guides/05_tabular_data_science_and_plots/using_gradio_with_bigquery.md

```diff
@@ -0,0 +1,110 @@
+# Using Gradio with Google BigQuery
+
+Google BigQuery is a cloud-based big data analytics web service for processing very large data sets. It is a serverless and highly scalable data warehousing solution that enables users to analyze data using SQL-like queries.
+
+In this tutorial, we will show you how to query a BigQuery dataset in Python and display the data *real-time* in a  dashboard using `gradio`. 
+
+We'll be working with the New York Times' COVID dataset that is available on BigQuery. The dataset, named `covid19_nyt.us_counties` contains the latest information about the number of confirmed cases and deaths from COVID across US counties. 
+
+**Prerequisites**: This Guide uses [Gradio Blocks](../01_getting_started/01_quickstart.md#blocks-more-flexibility-and-control), so please familiarize yourself with the Blocks class. 
+
+## Setting up your BigQuery Credentials
+
+To use Gradio with BigQuery, you will need to obtain your BigQuery credentials and use them with the BigQuery Python client. If you already have BigQuery credentials, you can skip this section. If not, you can do this for free in just a couple of minutes:
```

Comment:
Link to bigquery python client

Reply:
Can you add a title to this bulleted list, after the description, that's simpler? 

> How to Set Up Google BigQuery: 
> 1. First, log in to .....

Reply:
Linked the client, but skipped the title to bulleted list as it looked weird